### ISSUE TITLE
Staging Review finding: Confusing heading hierarchy for additional links section and sidebar

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#90541]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vfep-codereviewers
**Product:** Resources for schools
**Feature:** Convert Legacy Markdown SCO page into a React App

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content with errors or inconsistencies.](https://design.va.gov/about/experience-standards/#user-encounters-accurate-consistent-content)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

- The `Understanding Veteran education benefits` section uses an h3 heading, making it a subheading under `Other resources for schools`.
- The `Access Enrollment Manager` in the sidebar uses an h4 heading, making it a subheader under `Understanding Veteran education benefits`
- The sidebar headings after `Access Enrollment Manager` also use h3's, making them appear to be subsections under 'Other resources for schools` rather than a distinct group.

This doesn't clearly communicate the content hierarchy of the page and makes navigating by header more confusing for screen reader users.

<details><summary>Link, screenshot or steps to recreate</summary>

![headingi-hierarchy](https://github.com/user-attachments/assets/c5af350d-d988-439d-b05a-3462da7f0748)

</details>

## Recommended action

Review the heading hierarchy of the `Understanding Veteran education benefits` content and the sidebar to determine if they are subheadings for `Other resources for schools`, or if headings should be bumped up to h2's.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG21/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Page titles and section titles](https://design.va.gov/content-style-guide/page-titles-and-section-titles)
- **Context:** Headings

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by alexisbetz** [comment]
https://jira.devops.va.gov/browse/VEBT-419
